explorer/lib/explorer_web/live/utils.ex

```diff
@@ -123,12 +123,21 @@ defmodule Utils do
     |> Enum.reverse()
   end
 
-  def extract_amount_of_proofs_from_json({:ok, batch_json}) do
-    batch_json |> Enum.count()
+  def calculate_proof_hashes({:ok, batch_json}) do
+    IO.inspect("Calculating proof hashes")
+    batch_json
+      |> Enum.map(
+        fn proof ->
+          :crypto.hash(:sha3_256, proof["proof"])
+            # TODO removed this because i want to store as bytea, not a string.
```

Comment:
Is this TODO solved?

Reply:
This is already done in the frontend to show the proofs so yes, it is resolved. I'll delete the commented code